packages/odig/odig.0.0.8/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+synopsis: "Lookup documentation of installed OCaml packages"
+description: """\
+odig is a command line tool to lookup documentation of installed OCaml
+packages. It shows package metadata, readmes, change logs, licenses,
+cross-referenced `odoc` API documentation and manuals.
+
+odig is distributed under the ISC license. The theme fonts have their
+own [licenses](LICENSE.md).
+
+Homepage: https://erratique.ch/software/odig"""
+maintainer: "Daniel Bünzli <daniel.buenzl i@erratique.ch>"
+authors: "The odig programmers"
+license: [
+  "ISC" "LicenseRef-ParaType-Free-Font-License" "LicenseRef-DejaVu-fonts"
+]
+tags: [
+  "build" "dev" "doc" "meta" "packaging" "org:erratique" "org:b0-system"
+]
+homepage: "https://erratique.ch/software/odig"
+doc: "https://erratique.ch/software/odig/doc"
+bug-reports: "https://github.com/b0-system/odig/issues"
+depends: [
+  "ocaml" {>= "4.08"}
+  "ocamlfind" {build}
+  "ocamlbuild" {build}
+  "topkg" {build & >= "1.0.3"}
+  "cmdliner" {>= "1.1.0"}
+  "odoc" {>= "2.0.0"}
+  "b0" {= "0.0.4"}
```

Comment:
Any chance it could be `>=` instead? Why is the reason for such a tight constraint?

Reply:
Because I already know thed next version of `b0` will break `odig`.

Reply:
fair enough